packages/core/src/modules/dids/methods/peer/createPeerDidDocumentFromServices.ts

```diff
@@ -1,4 +1,4 @@
-import type { ResolvedDidCommService } from '../../../didcomm'
+import type { ResolvedDidCommService } from '../../../../types'
```

Comment:
I think it makes sense to keep this type in Core, and import it from the didcomm module (does not have to be done in this PR).

Because we support DidComm service for Did document, and thus I think it's ok to make this part of core